public/index.html

```diff
@@ -5,6 +5,18 @@
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <title>Cổng đăng ký FYT Code Cup</title>
+  <style>
+    @import url('https://fonts.googleapis.com/css?family=Mulish&display=swap');
+    body {
+      background-color: #F5F8FA;
+      margin: 0px;
+      margin-top: 2px;
+      padding: 0px;
+      font-family: Mulish, sans-serif;
+      font-style: normal;
+
+    }
+  </style>
```

Comment:
You can leave the import font here but lets move the style inside the `App.js` level (at `src/app/App.js`) like this:  

![image](https://user-images.githubusercontent.com/43344276/94437785-d39c6700-01d0-11eb-9b69-74c0abf20a57.png)

And the file `style.scss`:
```js
:local {
  body {
    background-color: #f5f8fa;
    margin: 0px;
    margin-top: 2px;
    padding: 0px;
    font-family: Mulish, sans-serif;
    font-style: normal;
  }
}
```